app/scrapers/parsijoo.py

```diff
@@ -88,7 +88,8 @@ def parse_news_response(soup):
             title = div.a.getText()
             link = unquote(div.a.get('href'))
             urls.append({'title': title, 'link': link})
-
-        print('Parsijoo parsed: ' + str(urls))
-
+        try:
+            print('Parsijoo parsed: ' + str(urls))
```

Comment:
If I don't use `try` `except` block, I am getting error : 
` 'charmap' codec can't encode characters in position 29-31: character maps to <undefined> `.
This error comes up when urls are empty.
It seems to be a console error rather than python as per [statckoverflow](https://stackoverflow.com/questions/32382686/unicodeencodeerror-charmap-codec-cant-encode-character-u2010-character-m) . So handled this way.

Reply:
looks like encoding problem